global_translations.R

```diff
@@ -38,3 +38,34 @@ complete_trans_df <- subset(final_df, untranslated_count == 0, select = -c(fuzzy
 
 # Untranslated Packages with respective languages associated
 complete_untrans_df <- subset(final_df, translated_count == 0, select = -c(fuzzy_count))
+
+# R Language Translation Team Contact details
+library(rvest)
+library(xml2)
+library(stringr)
+
+trans_team_wbpg <- read_html("https://developer.r-project.org/TranslationTeams.html")
```

Comment:
recommend using `rvest::html_table()` for this:

```r
translation_teams <- read_html("https://developer.r-project.org/TranslationTeams.html") %>%
  html_node("table") %>%
  html_table()
# A tibble: 21 × 2
   Language              Contact                                        
   <chr>                 <chr>                                          
 1 Chinese (Simplified)  "No current maintainer"                        
 2 Chinese (Traditional) "Wei-Lun Chao <chaoweilun@gmail.com>"          
 3 Danish                "Joe Dalton <joedalton2@yahoo.dk>"             
 4 Dutch                 "No current maintainer"                        
 5 French                "Philippe Grosjean\n <phgrosjean@sciviews.org>"
 6 German                "Detlef Steuer <steuer@hsu-hh.de>"             
 7 Greek (modern)        "Angelos I. Markos\n    <amarkos@gmail.com>"   
 8 Hungarian             "Daniel Kehl <kehld@ktk.pte.hu>"               
 9 Italian               "Daniele Medri <dmedri@gmail.com>"             
10 Japanese              "Masafumi Okada <mf@okadajp.org>"              
# … with 11 more rows
```

once the data is in a `data.frame` it should be easier to work with

Reply:
Hi Michael
I have made the required changes. Kindly review it.
Thanks!!